<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabRobotStatus">
       <attribute name="title">
        <string>Robot status</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lKErrorGlobal">
          <property name="text">
           <string>Error:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lVErrorGlobal">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Robot status</string>
          </property>
          <layout class="QGridLayout" name="layoutStatusRobot" columnstretch="0,1">
           <item row="9" column="0">
            <widget class="QLabel" name="lKHoldExt">
             <property name="text">
              <string>In hold by external</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lKCycle1">
             <property name="text">
              <string>1 cycle</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lKCommandRemote">
             <property name="text">
              <string>Command remote</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lKError">
             <property name="text">
              <string>Error</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lKAC">
             <property name="text">
              <string>Automatic and continuous</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="lKIGSO">
             <property name="text">
              <string>In guard safe operation</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="lKHoldCmd">
             <property name="text">
              <string>In hold by command</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lKAlarming">
             <property name="text">
              <string>Alarming</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="lKHoldPP">
             <property name="text">
              <string>In hold by PP</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="lKRunning">
             <property name="text">
              <string>Running</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lKServo">
             <property name="text">
              <string>Servo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="lKPlay">
             <property name="text">
              <string>Play</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="lKStep">
             <property name="text">
              <string>Step</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="lKTeach">
             <property name="text">
              <string>Teach</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lVServo">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lVAlarming">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="lVAC">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="lVCommandRemote">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="lVCycle1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="lVError">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="lVIGSO">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="lVHoldPP">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="lVHoldCmd">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="lVHoldExt">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="lVPlay">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLabel" name="lVRunning">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="lVStep">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QLabel" name="lVTeach">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabManualControl">
       <attribute name="title">
        <string>Manual control</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="grpCurrentPosition">
          <property name="title">
           <string>Current position</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radioPositionPulse">
               <property name="text">
                <string>Pulse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioPositionBaseCartesian">
               <property name="text">
                <string>Base cartesian</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioPositionUserCartesian">
               <property name="text">
                <string>User cartesian</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
